core/src/tangram.cpp

```diff
@@ -257,6 +257,11 @@ void Map::resize(int _newWidth, int _newHeight) {
 
 bool Map::update(float _dt) {
 
+    // Wait until font resources are fully loaded
+    if (impl->scene->fontContext()->resourceLoad > 0) {
```

Comment:
Maybe make sense to do this in render` call for texture resources, since these are not required for "tile building" which happens in the update loop.